platform/vs/docker-sonic-vs/start.sh

```diff
@@ -75,7 +75,6 @@ supervisorctl start redis-server
 start_chassis_db=`sonic-cfggen -v DEVICE_METADATA.localhost.start_chassis_db -y $chassisdb_cfg_file`
 if [[ "$HOSTNAME" == *"supervisor"* ]] || [ "$start_chassis_db" == "1" ]; then
    supervisorctl start redis-chassis
-   python /usr/bin/chassis_db.py
```

Comment:
now that these files are no longer required, can the files chassis_db.py and chassis_db.json be removed ?

Reply:
Yes. I'll remove them

Reply:
Removed chassis_db.py here and chassis_db.json from https://github.com/Azure/sonic-swss/pull/1431